Libraries/Components/Touchable/Touchable.js

```diff
@@ -813,12 +869,12 @@ const TouchableMixin = {
     UIManager.playTouchSound();
   },
 
-  _startHighlight: function(e) {
+  _startHighlight: function(e: LayoutEvent) {
     this._savePressInLocation(e);
     this.touchableHandleActivePressIn && this.touchableHandleActivePressIn(e);
   },
 
-  _endHighlight: function(e) {
+  _endHighlight: function(e: LayoutEvent) {
```

Comment:
This method calls `this.touchableHandleActivePressOut`, which you typed as accepting a `PressEvent` inside `TouchableNativeFeedback.android.js`. So, shouldn't the type of `e` also be `PressEvent`?